unittests/test_utility.py

```diff
@@ -187,13 +187,17 @@ def test_is_url():
     assert not os_ext.is_url(repo_ssh)
 
 
-def test_git_repo_hash():
-    # A git branch hash consists of 8(short) or 40 characters.
+def test_git_repo_hash(monkeypatch):
+    # a git branch hash consists of 8(short) or 40 characters.
```

Comment:
I think our style is to start comments with a capital letter unless the comment is inline.